Formula/xcdiff.rb

```diff
@@ -0,0 +1,34 @@
+class Xcdiff < Formula
+  desc "Tool to diff xcodeproj files"
+  homepage "https://github.com/bloomberg/xcdiff"
+  url "https://github.com/bloomberg/xcdiff.git",
+    tag:      "0.10.0",
+    revision: "289872e572224ae429d0f4d25ff9be906c9df1a0"
+  license "Apache-2.0"
+  head "https://github.com/bloomberg/xcdiff.git", branch: "main"
+  depends_on :macos
+  depends_on xcode: "14.1"
+
+  resource "homebrew-testdata" do
+    url "https://github.com/bloomberg/xcdiff/archive/refs/tags/0.10.0.tar.gz"
+    sha256 "c093e128873f1bb2605b14bf9100c5ad7855be17b14f2cad36668153110b1265"
+  end
+
+  def install
+    system "make", "update_version"
+    system "make", "update_hash"
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+    bin.install ".build/release/xcdiff"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/xcdiff --version").chomp
+    project = "Fixtures/ios_project_1/Project.xcodeproj"
+    diff_args = "-p1 #{project} -p2 #{project}"
+    resource("homebrew-testdata").stage do
+      assert_match "\n", shell_output("#{bin}/xcdiff #{diff_args} -d")
```

Comment:
Can we assert something more than a linebreak here?

Reply:
This is the case when there is no diff as such breakline is the only output. One possible improvement would be to assert also the exit code, but I wasn't sure if there is a way to capture it when using `shell_output`.

Reply:
If it's only that, this is the better option
```suggestion
     # assert no difference between projects
      assert_equal "\n", shell_output("#{bin}/xcdiff #{diff_args} -d")
```